Add endpoints docs to wiki (#883)
iota-wiki/iota-wiki: next/develop/endpoints/testnet.md

---
description: The IOTA foundation provides load-balanced public testnet endpoints, where MQTT and the HTTP REST API are enabled.
image: /img/logo/preview.png
keywords:
  - devnet
  - load-balanced
  - HTTP REST API
  - MQTT
  - reference
  - Endpoints
---

# Public Testnet

The testnet is the public infrastructure for developers. It runs on top of the beta releases of IOTA software and runs a [faucet](https://faucet.testnet.shimmer.network) to easily get tokens for testing.

## Public Infrastructure

The IOTA Foundation currently provides the following public load-balanced testnet endpoints:

- Node API: https://api.testnet.shimmer.network
  - Example info endpoint: https://api.testnet.shimmer.network/api/core/v2/info
  - Available routes: https://api.testnet.shimmer.network/api/routes
  - Health endpoint: https://api.testnet.shimmer.network/health
- MQTT: wss://api.testnet.shimmer.network:443/api/mqtt/v1
- Faucet Info API: https://faucet.testnet.shimmer.network/api/info
- Faucet Enqueue API: https://faucet.testnet.shimmer.network/api/enqueue
- Chronicle API: https://chronicle.testnet.shimmer.network

> These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this specifications [TIP-25](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0025/core-rest-api.yaml), [TIP-26](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0026/indexer-rest-api.yaml) over TLS.

Reviewer: ```suggestion
These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this specifications [TIP-25](nodes/rest-api/iota-rest-api.info.mdx), [TIP-26](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0026/indexer-rest-api.yaml) over TLS.
```

This one too probably?

---

> description: The IOTA foundation provides load-balanced public Shimmer Beta endpoints, where MQTT and the HTTP REST API are enabled.

Author: ```suggestion
description: The IOTA foundation provides load-balanced public testnet endpoints, where MQTT and the HTTP REST API are enabled.
```

image: /img/logo/preview.png
keywords:
- devnet
- load-balanced
- HTTP REST API
- MQTT
- reference
- Endpoints
---

> # Shimmer Beta
> 
> Shimmer Beta is a pre-release of the Shimmer network that is currently under development.

Author: ```suggestion
# Testnet

Testnet is a pre-release of the Shimmer network that is currently under development.
```

## Public Infrastructure

> We currently provide load-balanced public Shimmer Beta endpoints:

Author: ```suggestion
We currently provide load-balanced public testnet endpoints:
```

 - Node API: https://api.testnet.shimmer.network
   - Example info endpoint: https://api.testnet.shimmer.network/api/core/v2/info
   - Available routes: https://api.testnet.shimmer.network/api/routes
   - Health endpoint: https://api.testnet.shimmer.network/health
 - MQTT: wss://api.testnet.shimmer.network:443/api/mqtt/v1
 - Chronicle API: https://chronicle.testnet.shimmer.network

These endpoints have MQTT (via WebSockets and raw TCP) exposed and offer the HTTP REST API (according to this specifications [TIP-25](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0025/core-rest-api.yaml), [TIP-26](https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/main/tips/TIP-0026/indexer-rest-api.yaml) over TLS.

## Developer Tools

- [Explorer](https://explorer.shimmer.network)
- [Online Faucet](https://faucet.testnet.shimmer.network)
- Faucet Info API: https://faucet.testnet.shimmer.network/api/info
- Faucet Enqueue API: https://faucet.testnet.shimmer.network/api/enqueue

> - [Cli-Wallet](https://github.com/iotaledger/cli-wallet/tree/develop)
> - [shimmer-faucet code (nodejs + svelte)](https://github.com/iotaledger/chrysalis-faucet/tree/hornet)
> - [shimmer-faucet backend](https://github.com/iotaledger/inx-faucet)

Author: Same as above

```suggestion
```
